build_quarto_pdfs.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+"""
+build pdf version for specified MAJOR, MINOR version and specified document (optional)
+when document not specified, builds all documents
+
+arg 1: MAJOR version number
+arg 2: MINOR version number
+optional arg 3: specified document or "all"
+"""
+
+import glob
+import os
+import os.path
+import shutil
+import subprocess
+import sys
+import contextlib
+
+all_docs = ("functions-reference", "reference-manual", "stan-users-guide", "cmdstan-guide")
+
+@contextlib.contextmanager
+def pushd(new_dir):
+    previous_dir = os.getcwd()
+    os.chdir(new_dir)
+    yield
+    os.chdir(previous_dir)
+
+def shexec(command):
+    ret = subprocess.run(command, shell=True, check=False, stdout=subprocess.PIPE, stderr=subprocess.PIPE, universal_newlines=True)
+    if ret.returncode != 0:
+        print('Command {} failed'.format(command))
+        print(ret.stderr)
+        raise Exception(ret.returncode)
+
+def safe_rm(fname):
+    if os.path.exists(fname):
+        os.remove(fname)
+
+def make_docs(docspath, version, document):
+    path = os.getcwd()
+    srcpath = os.path.join(path, "src", document)
+    with pushd(srcpath):
+        pdfname = ''.join([document,'-',version,".pdf"])
+        print('render {} as {}'.format(document, pdfname))
+        command = ' '.join(['quarto render',
+                                '--output-dir _pdf',
+                                '--output', pdfname]) 
+        shexec(command)
+        outpath = os.path.join(srcpath, '_pdf', pdfname)
+        shutil.move(outpath, docspath)
+        safe_rm(".gitignore")  # quarto cleanup
```

Comment:
Rather than delete this I think we should just check-in .gitignores in each of these folders. We can also ignore `_pdf/` and `_website/` in the top-level .gitignore